### ISSUE TITLE
[csrng] prim_count error not hooked up to alert

### ISSUE THREAD

**Issue by tjaychen** [lowRISC/opentitan#9083]
While working on #9082, it was discovered that the prim_count error does not appear to be hooked up to either a recoverable or fatal event.

@mwbranstad can you double check and let me know if I just misread?

**Comment by mwbranstad** [comment]
@tjaychen you are correct - the status bit was connected, but not the actual fatal_err pin. PR in progress.
